frontend/src/NavBar/Greeting.js

```diff
@@ -49,15 +164,30 @@ const Greeting = React.memo(({ currentUser, logout }) => {
           <SecondaryButton onClick={handleLoginClick}>Log In</SecondaryButton>
         </div>
       ) : (
-        <div>
-          <GreetingAndLogoutDiv>
-            <StyledAvatar src={currentUser.photoUrl} alt="" />
-            <GreetingText>Hi, {currentUser.username}!</GreetingText>
-            <LogoutButton type="submit" onClick={logout}>
-              Log Out
-            </LogoutButton>
-          </GreetingAndLogoutDiv>
-        </div>
+        <Container>
+          <MenuTrigger type="button" onClick={onClick}>
+            <AvatarImg src={currentUser.photoUrl} />
+          </MenuTrigger>
+
+          <Dropdown isActive={isActive} ref={dropdownMenuRef}>
+            <Ul>
+              <Item>
+                <Link href="/#/restaurants/new">CREATE RESTAURANT</Link>
+              </Item>
+              <Item>
+                <Link href="/#/account/reservations">RESERVATIONS</Link>
+              </Item>
+              <Item>
+                <Link href="/#/account/favorites">FAVORITES</Link>
```

Comment:
These shouldn't need the `/#` prefix

Reply:
Also, same comment as elsewhere about these having normal casing + text transform.